pkg/security/module/event_easyjson.go

```diff
@@ -1,158 +1,19 @@
-// Code generated by easyjson for marshaling/unmarshaling. DO NOT EDIT.
+// TEMPORARY AUTOGENERATED FILE: easyjson stub code to make the package
```

Comment:
I think the generation was not over when this was commited

Reply:
oops